wls-vorfaelleundvorkommnisse-service/src/test/java/de/muenchen/oss/wahllokalsystem/vorfaelleundvorkommnisseservice/rest/ereignis/EreignisDTOMapperTest.java

```diff
@@ -0,0 +1,46 @@
+package de.muenchen.oss.wahllokalsystem.vorfaelleundvorkommnisseservice.rest.ereignis;
+
+import de.muenchen.oss.wahllokalsystem.vorfaelleundvorkommnisseservice.domain.ereignis.Ereignisart;
+import de.muenchen.oss.wahllokalsystem.vorfaelleundvorkommnisseservice.service.EreignisModel;
+import de.muenchen.oss.wahllokalsystem.vorfaelleundvorkommnisseservice.utils.TestdataFactory;
+import java.util.ArrayList;
+import java.util.List;
+import lombok.val;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.mapstruct.factory.Mappers;
+
+class EreignisDTOMapperTest {
```

Comment:
Vorschlag: Auch wenn es nicht zu jeder Methode mehrere Testfälle gibt, könnte man die Method doch über eine `@Nested`-Annotiertes Klasse als Testsuit klar erkennbar machen

Reply:
siehe commit [625b376](https://github.com/it-at-m/Wahllokalsystem/pull/412/commits/625b376350bd2305f26834050bb0c61d424dd9b0)